tests/features/createEnvRegistry.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Support multiple configuration providers in Env. Registry
+
+Scenario: Create EnvRegistry with available configProvider and valid token
+  Given a configuration
+  And token "tokenA" that allows access to this configuration using <configProvider> and <dispatcherUrl>
```

Comment:
I'm not sure, but maybe it makes sense to provide more realistic token value?
This one is ok only for LocalFileConfigurationProvider.

You can check this page to take a look at an examples for tokens for each provider
https://github.com/capsulajs/configuration-service